### ISSUE TITLE
Build fails if not able to sent the metrics

### ISSUE THREAD

**Issue by stepango** [cdsap/Talaiot#86]
```
Exception in thread “pool-5-thread-1” org.influxdb.InfluxDBException: partial write: max-series-per-database limit exceeded: (1000000) dropped=2955
    at org.influxdb.InfluxDBException.buildExceptionFromErrorMessage(InfluxDBException.java:161)
    at org.influxdb.InfluxDBException.buildExceptionForErrorState(InfluxDBException.java:173)
    at org.influxdb.impl.InfluxDBImpl.execute(InfluxDBImpl.java:808)
    at org.influxdb.impl.InfluxDBImpl.write(InfluxDBImpl.java:463)
    at com.cdsap.talaiot.publisher.InfluxDbPublisher$publish$1.run(InfluxDbPublisher.kt:74)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
    at java.lang.Thread.run(Thread.java:748)
```

**Comment by cdsap** [comment]
hi @stepango, thanks for the report. I'll take a look

**Comment by cdsap** [comment]
merged fix to catch the exception:

<img width="941" alt="Screenshot 2019-08-01 at 23 15 20" src="https://user-images.githubusercontent.com/475733/62312299-0907d080-b4b8-11e9-9846-8b86c6ed4053.png">


Published snapshot 1.0.3-SNAPSHOT

**Comment by cdsap** [comment]
last night was published 1.0.3 release version with the fix

**Comment by cdsap** [comment]
Hi @stepango, can I close the issue? Is fixed since 1.0.3?

**Comment by stepango** [comment]
Hi, yes. Issue is fixed
